ui/src/app/store/user/selectors.ts

```diff
@@ -60,11 +60,23 @@ const markingIntroCompleteErrors = createSelector(
     eventErrors.find((eventError) => eventError.event === "markIntroComplete")
       ?.error || null
 );
+
+/**
+ * Get the auth user superuser status.
+ * @param state - The redux state.
+ * @returns {Bool} Is auth user superuser.
+ */
+const isSuperUser = createSelector(
```

Comment:
We actually happen to have an `isAdmin` selector that does the same thing: https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/store/auth/selectors.ts#L54